packages/web/components/Landing/WhatSay.tsx

```diff
@@ -0,0 +1,173 @@
+          flex="1"
+          background="linear-gradient(180deg, rgba(23, 11, 41, 0.00) 0%, #170B28 100%)"
+          pos="absolute"
+          h="70%"
+          bottom={-9}
+          w="100vw"
+          zIndex={100}
+          pointerEvents="none"
+        />
+        <Flex
+          zIndex={99}
+          gap={20}
+          {...{ ref }}
+          display="flex"
+          flexDirection="column"
+          justifyContent={{ base: 'flex-start', xl: 'center' }}
+          maxWidth={{
+            base: '90%',
+            md: '3xl',
+            xl: '5xl',
+            '2xl': '8xl',
+            '4xl': '100%',
+          }}
+          maxH={{ base: '60%', md: '90%', lg: 'initial' }}
+          overflowY={{ base: 'auto', xl: 'visible' }}
+          pl={{ base: 0, md: 0 }}
+          transform={`translate3d(0, ${displayElement ? '0' : '50px'}, 0)`}
+          opacity={displayElement ? 1 : 0}
+          transition={
+            'transform 0.3s 0.1s ease-in-out, opacity 0.5s 0.2s ease-in'
+          }
+          sx={{
+            h2: {
+              color: 'landing500',
+              fontSize: { base: 'xl', md: '3xl', xl: '3xl', '2xl': '4xl' },
+              textAlign: 'center',
+              textShadow: '0 0 5px var(--chakra-colors-landing500)',
+              transform: {
+                base: 'none',
+                xl: 'translate3d(0, 15px, 0)',
+                '4xl': 'translate3d(0, -60px, 0)',
+              },
+            },
+          }}
+        >
+          <Text as="h2" fontWeight="700">
+            What people are saying
+          </Text>
+
+          <Flex gap={8}>
+            <Flex flex="1" direction="column" gap={8}>
+              <WhatSayCard
+                player="METADREAMER"
+                testimony="MetaGame was my gateway drug to web3. It felt like I was reading my own internal thoughts on how the world could be, and to this day it’s my guiding philosophy for the future of society. Being able to collaborate with people who felt the same to help build it was the best way to start contributing in web3 and became a launchpad / network to a bunch of the best projects and people in the space"
+                username="metadreamer"
+              />
+
+              <WhatSayCard
+                player="Kyle Stargarden"
+                testimony="MetaGame was the launching pad for my career in Web3. When I joined Metagame I was working as a personal assistant. Now I am Lead Blockchain Engineer and public speaker at some of the world's top Web3 conferences."
+                username="kyle-stargarden"
+              />
+            </Flex>
+            <Flex flex="1" direction="column" gap={8}>
+              <WhatSayCard
+                player="Yalor"
+                testimony="MetaGame inspired me join the Web3 ecosystem after a season of disillusionment and ICO scams 😛
+      Through MetaGame I found the most amazing community of values aligned ecosystem innovators that I still work with today 🤩"
```

Comment:
Is this valid syntax for this component? We're going to lose the paragraphs on rendering.

```suggestion
                testimony={
                  <Text>MetaGame inspired me join the Web3 ecosystem after a season of disillusionment and ICO scams 😛</Text>
                  <Text>Through MetaGame I found the most amazing community of values aligned ecosystem innovators that I still work with today 🤩<Text>
                }
```